manifests/4.2/local-storage-operator.v4.2.0.clusterserviceversion.yaml

```diff
@@ -29,11 +29,12 @@ metadata:
       ]
     categories: Storage
     capabilities: Full Lifecycle
-    containerImage: quay.io/openshift/origin-local-storage-operator:4.2.0
+    containerImage: quay.io/openshift/origin-local-storage-operator:latest
     support: Red Hat
     repository: https://github.com/openshift/local-storage-operator
     createdAt: "2019-08-14T00:00:00Z"
     description: Configure and use local storage volumes in kubernetes and Openshift
+    olm.skipRange: '>=4.2.0 <4.3.0'
```

Comment:
A 4.2 manifest in 4.3 branch should have no effect IMO on build process. It is only kept for keeping build scripts happy when transition between branches happen.

Reply:
Understood. I can remove this reference here.